### DISCUSSION TITLE
A more versatile image registration process

### DISCUSSION THREAD

**Discussion by yuGithuuub** [IBEXImagingCommunity/ibex_imaging_knowledge_base#85] [General]
Dear authors
  Thank you for providing such a simple multiplex immunofluorescence method.
  In practical application, I encountered some problems.
  Our laboratory does not have a thunder microscope, but uses Olympus VS200, 3DHISTECH or AKOYA scanner for imaging.
  The output data formats of these companies cannot be well converted into ims files.
  Is there a more general image registration solution for ome.tiff files and the like?
Looking forward to your reply
Best wishes！
Simon

**Comment by zivy** [comment]
Hello @yuGithuuub,

Unfortunately, we do not have plans for supporting ome-tiff. We do plan to support the ome-zarr format though. 

What specifically was the problem you encountered when converting your images to the ims format? Can you provide some details?

If you can convert the files to ome-tiff it should be possible to convert them to ims format. For now, if you have them in ome-tiff format, possibly try this [tif2ims workaround](https://github.com/zivy/tif2ims). The code does more than conversion, but can be used to convert from tif2ims and ims2tif. 

Let us know if this workaround resolves your issue. If not, possibly share a sample ome-tiff file that is problematic and describe the issue with the specific file in detail and we will try to figure out how to convert it to ims format.
